src/v2/controllers/vreplicaset_controller/proof/liveness/resource_match.rs

```diff
@@ -598,6 +605,7 @@ pub proof fn lemma_from_after_receive_list_pods_resp_to_send_create_pod_req(
     );
 }
 
+#[verifier(rlimit(10000))]
```

Comment:
Maybe also add a todo here. Does this one really need so large rlimit?

Reply:
They seem not to require extra `rlimit` now. Nevertheless, I'm marking them as flaky proofs.